clifford/_layout.py

```diff
@@ -353,18 +353,6 @@ def __init__(self, *args, **kw):
                 "names list of length %i needs to be of length %i" %
                 (len(names), self.gaDims))
 
-        # preload these lazy properties. Not doing this would likely be faster.
-        self.gmt_func
-        self.imt_func
-        self.omt_func
-        self.lcmt_func
-        self.adjoint_func
-        self.left_complement_func
-        self.right_complement_func
-        self.dual_func
-        self.vee_func
-        self.inv_func
```

Comment:
I worry this change will have made this benchmark meaningless:

https://github.com/pygae/clifford/blob/ce0bc74520451a05ab156bd07ce3db783b4f0839/clifford/test/test_algebra_initialisation.py#L14-L22

Can you add these lines to the benchmark?